daml-lf/spec/daml-lf-2.rst

```diff
@@ -4116,83 +4106,52 @@ Numeric functions
   inputs and the output is given by the type parameter `α`.  Throws an
   ``ArithmeticError`` exception in case of overflow.
 
-* ``MUL_NUMERIC_LEGACY : ∀ (α₁ α₂ α : nat) . 'Numeric' α₁ → 'Numeric' α₂ → 'Numeric' α``
+* ``MUL_NUMERIC : ∀ (α₁ α₂ α : nat) . 'Numeric' α → 'Numeric' α₁ → 'Numeric' α₂ → 'Numeric' α``
 
   Multiplies the two numerics and rounds the result to the closest
   multiple of ``10⁻ᵅ`` using `banker's rounding convention
   <https://en.wikipedia.org/wiki/Rounding#Round_half_to_even>`_.  The
   type parameters `α₁`, `α₂`, `α` define the scale of the first input,
-  the second input, and the output, respectively.  Throws an
-  ``ArithmeticError`` exception in case of overflow.
+  the second input, and the output, respectively.  The first argument
+  is used as a witness for the scale `α`, is actual value is ignored.
```

Comment:
```suggestion
  is used as a witness for the scale `α`, its actual value is ignored.
```

Reply:
Is its value actually ignored? I thought the value was providing the scale.

Reply:
```suggestion
  is used as a witness for the scale `α`.
```